SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# **Security Vulnerability Management Process for SONiC Community**
+This document outlines SONiC vulnerability reporting and management process. SONiC is a popular choice of cloud providers, enterprises, telecom providers, web service providers and others to build their digital infrastructure.  The security of SONiC is vital for the safety and reliability of our digital transformation. The strong and active cooperation among SONiC community members is key to securing SONiC. This process will be shared through https://github.com/sonic-net/SONiC/SECURITY.md file after TSC approval.  The diagram below illustrates the high-level workflow:
+![](./images/security-process/security.png)
+## 1.Report SONiC Vulnerability
+As a SONiC community member, it is your responsibility to report the discovered vulnerabilities before public disclosure. If you want to report a vulnerability, please encrypt your email and privately send it to security@lists.sonicfoundation.dev. Only SONiC security committee members can access the information in this security mailing list, and they also watch over this mailing list.  When someone reports a new vulnerability, SONiC security committee will assist with the vulnerability assessment, coordinate on the mitigation and fix. If you have a suggested fix or mitigation, please include it in your report. Exploit instruction is very useful and will be kept confidential unless it is already public. SONiC security committee may seek help from SONiC repo maintainers or other domain experts to look into the vulnerability and prepare a mitigation/fix. The collaboration and communication will be private.
```

Comment:
A standard template will help set the expectation for submitter. A limited access defect tracking system (eg, JIRA) will keep things organized as the reported incident transition through stages of triage, reproduction, information gathering, root cause analysis, fix and verification, publication etc

Reply:
> A standard template will help set the expectation for submitter. A limited access defect tracking system (eg, JIRA) will keep things organized as the reported incident transition through stages of triage, reproduction, information gathering, root cause analysis, fix and verification, publication etc

Committee can define a security issue template to follow.